src/Assert.php

```diff
@@ -1393,9 +1498,11 @@ public static function maxLength($value, $max, $message = '')
     /**
      * Inclusive , so Assert::lengthBetween('asd', 3, 5); passes the assertion.
      *
-     * @param mixed  $value
-     * @param mixed  $min
-     * @param mixed  $max
+     * @psalm-pure
+     *
+     * @param string $value
+     * @param int    $min
```

Comment:
int|float would be more accurate here.

Reply:
This goes for other min/max params as well

Reply:
48a9778070e65542b0f9ca58803fbf59a26ec827